.evg.yml

```diff
@@ -3,100 +3,196 @@ functions:
     - command: git.get_project
       params:
         directory: "stitch-ios-sdk"
+  "setup_xcov":
+    - command: shell.exec
+      params:
+        shell: "bash"
+        script: |
+          gem install xcov
+  "setup_mongod":
+    - command: shell.exec
+      params:
+        shell: "bash"
+        script: |
+          set -e
+          curl --silent ${mongodb_url} | tar xvz
+    - command: shell.exec
+      params:
+        shell: "bash"
+        background: true
+        script: |
+          set -e
+          cd mongodb-*
+          echo "starting mongod..."
+          mkdir db_files
+          ./bin/mongod --dbpath ./db_files --port 26000
+    - command: shell.exec
+      params:
+        shell: "bash"
+        script: |
+          set -e
+          cd mongodb-*
+          echo "waiting for mongod to start up"
+          ./bin/mongo --nodb --eval 'assert.soon(function(x){try{var d = new Mongo("localhost:26000"); return true}catch(e){return false}}, "timed out connecting")'
+          echo "mongod is up."
 
-  "set_github_build_status":
+  "setup_stitch":
     - command: shell.exec
       params:
+        shell: "bash"
         silent: true
         script: |
-          status_url=`curl -X POST -d "{}" -H "Content-Type: application/json" "${stitch_evg_hook_url}?secret=${stitch_evg_secret}&versionId=${version_id}"`
-          if [ "$status_url" = "null" ]; then
-            exit 0
-          fi
-          status_url=`sed -e 's/^"//' -e 's/"$//' <<<"$status_url"`
-          state=`cat commit_state`
-          curl -X POST -H "Content-Type: application/json" $status_url -d "{\"state\": \"$state\", \"target_url\": \"https://evergreen.mongodb.com/build/${build_id}\", \"context\": \"continuous-integration/evergreen-ci/${build_variant}\"}"
+          set -e
+          echo export AWS_ACCESS_KEY_ID=${test_aws_key} >> creds
+          echo export AWS_SECRET_ACCESS_KEY=${test_aws_secret} >> creds
+    - command: shell.exec
+      params:
+        shell: "bash"
+        script: |
+          set -e
+          echo "cloning stitch"
+          mkdir -p src/github.com/10gen
+          git clone git@github.com:10gen/stitch src/github.com/10gen/stitch
 
+          echo "downloading libmongo"
+          wget --quiet "https://s3.amazonaws.com/stitch-artifacts/stitch-mongo-libs/stitch_mongo_libs_linux_64_patch_1f02b06ec6fd48bad0b725b77bc194c84f839f53_5ab7216fc9ec44724e08f194_18_03_25_04_11_28/libmongo.so"
```

Comment:
This is a linux shared object that isn't available. You want these two instead:
https://s3.amazonaws.com/stitch-artifacts/stitch-mongo-libs/stitch_mongo_libs_osx_patch_cbcbfd8ebefcca439ff2e4d99b022aedb0d61041_59e2b7a5c9ec4432c400181c_17_10_15_01_19_33/update_doc
https://s3.amazonaws.com/stitch-artifacts/stitch-mongo-libs/stitch_mongo_libs_osx_patch_cbcbfd8ebefcca439ff2e4d99b022aedb0d61041_59e2b7ab2a60ed5647001827_17_10_15_01_19_39/assisted_agg

both need to be chmod +x